GVFS/GVFS.Common/NuGetUpgrader/NuGetFeed.cs

```diff
@@ -128,6 +129,28 @@ public virtual async Task<string> DownloadPackageAsync(PackageIdentity packageId
             return downloadPath;
         }
 
+        public virtual bool VerifyPackage(string packagePath, string otherCertificateFingerprintToTrust)
+        {
+            List<string> certificateFingerprints = new List<string>();
+            certificateFingerprints.Add("3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE");
+            if (otherCertificateFingerprintToTrust != null)
+            {
+                certificateFingerprints.Add(otherCertificateFingerprintToTrust);
+            }
+
+            VerifyArgs verifyArgs = new VerifyArgs()
+            {
+                Verifications = new VerifyArgs.Verification[] { VerifyArgs.Verification.All },

```

Comment:
What does `VerifyArgs.Verification.All` mean?

Reply:
The documentation on this seems a bit murky... the [NuGet CLI](https://docs.microsoft.com/en-us/nuget/tools/cli-ref-verify) documentation indicates that "All possible verifications will be performed". It looks like the only verifications are "Signature" verifications at this point in time...

 See for [example](https://github.com/NuGet/NuGet.Client/blob/7ca842c4386e287ba456a115979a7946766509df/src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyCommandRunner.cs#L38)